pkg/frontend/adminactions/resources_list.go

```diff
@@ -76,3 +62,41 @@ func (a *adminactions) ResourcesList(ctx context.Context) ([]byte, error) {
 
 	return json.Marshal(armResources)
 }
+
+func (a *adminactions) appendAzureNetworkResources(ctx context.Context, armResources []arm.Resource, vNetID string, vnetClient network.VirtualNetworksClient, routeTablesClient network.RouteTablesClient) []arm.Resource {
+	r, err := azure.ParseResourceID(vNetID)
+	if err != nil {
+		a.log.Warnf("skipping vnet '%s' due to parse error: %s", vNetID, err)
```

Comment:
have this function return ([]arm.Resource, error) as usual and have the caller log and then ignore the error

Reply:
Changed to return an error, added an if to log+ignore in ResourcesList function.